src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -454,8 +407,56 @@ impl Interpreter for CopyInterpreter {
         }
 
         match &self.plan {
-            CopyPlan::IntoTable(plan) => self.build_copy_into_table_pipeline(plan).await,
-
+            CopyPlan::IntoTable(plan) => {
+                if plan.enable_distributed {
+                    let (distributed_plan_op, empty_files) =
+                        self.transform_copy_plan_distributed(plan).await?;
+                    if empty_files {
+                        return self.build_copy_into_table_pipeline(plan).await;
+                    }
+                    let distributed_plan = distributed_plan_op.unwrap();
```

Comment:
please doc why it is safe to unwrap here. besides,  it seems that `transform_copy_plan_distributed` returns either `(None, True)` or `(Some(_), False)`, in this case, can we just return `Result<Option<T>>` ?